### ISSUE TITLE
Support Namespace Import in Mappers

### ISSUE THREAD

**Issue by janhartmann** [dotansimha/graphql-code-generator#1825]
**Is your feature request related to a problem? Please describe.**

I am generating some types based on a Swagger Spec and because of some name collision we are now namespacing the types in e.g. 

```tsx
export namespace MachinesApiTypes {
   export interface OdometerRepresentation {
      distanceUnit?: "KM" | "MI";
      distance?: number;
    }
}
```

Now in my `codegen.yml` mapper section I need to map `MyApiTypes.OdometerRepresentation` to a type:

```yml
schema: "schema.graphql"
overwrite: true
generates:
 generated-types.ts:
    config:
      mappers:
        Odometer: ../api/machines/MachinesApiTypes#MachinesApiTypes.OdometerRepresentation
```

But this does not work as the import statement should only be the:

```ts
import { MachinesApiTypes } from "../../api/machines/MachinesApiTypes";
```

It is current imported as, and not working:

```ts
import { MachinesApiTypes.OdometerRepresentation } from "../../api/machines/MachinesApiTypes";
```

**Describe the solution you'd like**
I would like to be able to import a namespace and reference the types where applicable, maybe a configuration:

```yml
config:
      mappers:
        Odometer: ../api/machines/MachinesApiTypes#MachinesApiTypes#OdometerRepresentation
```

**Describe alternatives you've considered**
Not using namespaces, but we are generating a ton of code and name collisions will be hard in the long run.

**Comment by kamilkisiela** [comment]
I will talk with the team how they see this feature but for now you can use `@graphql-codegen/add` plugin and do the following:

```yaml
plugins:
  add: import { MachinesApiTypes } from '../../api/machines/MachinesApiTypes';
  ...
config:
  mappers:
    Odometer: MachinesApiTypes.OdometerRepresentation

```

**Comment by janhartmann** [comment]
Interesting approach, thanks!

**Comment by kamilkisiela** [comment]
#2352 should enable namespaces

**Comment by dotansimha** [comment]
Fixed in 1.6.0
